docs/tutorial/part03.rst

```diff
@@ -2,5 +2,245 @@
 How to create the base models.
 ==============================
 
-.. todo::
-   Here will be the tutorial body.
\ No newline at end of file
+Create a new app for the API: 
+==============================
+Right now, we need to create an app which is going to be use  to create the API and, with it, the models definition. If you do not know the definition of an API; it is “`code that allows two software programs to communicate with each other`”. 
+
+To create this new app, it is needed to run this command: 
+
+``python manage.py startapp fetch_api``
+
+Where ``fetch_api`` is the name of the application we are creating, if you prefer you can use any other name.  This will create a new folder inside the program we have, and the structure of it is: ::
+
+    ├── fetch_api/
+    │   ├── __init__.py
+    │   ├── admin.py
+    │   ├── apps.py
+    |   ├── models.py
+    │   ├── migrations/
+    │   │   └── __init__.py 
+
+
+Create The Models Directory:
+==============================
+
+On The ``fetch_api`` folder, first, we need to remove the ``models.py`` file. Then we need to create a new folder called ``models``.  Inside the folder, you will need to add the files with the structure of the nodes. Each file will have the essential fields and methods of the data stored in Neo4j database. 
+
+.. [*] Note: Before we start creating our models we need to create a  ``__init__.py`` file inside the model’s folder so Python treat the directory as a module

```

Comment:
Amend, it should be:
"treat the directory as a *package*"

Also, add a dot at the end.

Reply:
you are right it is a package